MachineArithmetic-Tests/ContextlessZ3Test.class.st

```diff
@@ -59,10 +59,11 @@ ContextlessZ3Test >> testSymbolFromString [
 { #category : #tests }
 ContextlessZ3Test >> testTwoContexts [
 	"Several logical context can be used simultaneously."
-	| ctx1 ctx2 x y str |
+
+	| ctx1 ctx2 x y |
 	ctx1 := Z3Context fromDefault.
 	ctx2 := Z3Context fromDefault.
-	self deny: ctx1 handle asInteger = ctx2 handle asInteger.
+	self deny: ctx1 getHandle asInteger equals: ctx2 getHandle asInteger.
```

Comment:
There's no `#deny:equals:` in St/X (as yet) - this is why test on St/X fails. 
Could you please change it like:
```
diff --git a/MachineArithmetic-Tests/ContextlessZ3Test.class.st b/MachineArithmetic-Tests/ContextlessZ3Test.class.st
index 114c32d..268e0a9 100644
--- a/MachineArithmetic-Tests/ContextlessZ3Test.class.st
+++ b/MachineArithmetic-Tests/ContextlessZ3Test.class.st
@@ -63,7 +63,7 @@ ContextlessZ3Test >> testTwoContexts [
        | ctx1 ctx2 x y |
        ctx1 := Z3Context fromDefault.
        ctx2 := Z3Context fromDefault.
-       self deny: ctx1 getHandle asInteger equals: ctx2 getHandle asInteger.
+       self deny: ctx1 getHandle asInteger = ctx2 getHandle asInteger.
        x := ctx1 mkBoolVar: 'x'.
        y := ctx2 mkBoolVar: 'y'.
        ctx1 del.
```
I just tested the above change in St/X